.gitignore

```diff
@@ -20,6 +20,7 @@
 !\.yapfignore
 !\.zuul
 !\.zuul.yaml
+!\.github/workflows/*
```

Comment:
please dont do this - rather clean up the yaml

Reply:
it's not necessary but it makes it a lot less annoying to change workflow files as it means git add * will add them.